src/libpgmoneta/utils.c

```diff
@@ -4004,6 +4005,290 @@ pgmoneta_indent(char* str, char* tag, int indent)
+
+   if (header)
+   {
+      command = (int32_t)pgmoneta_json_get(header, MANAGEMENT_ARGUMENT_COMMAND);
+      translated_command = translate_command(command);
+      if (translated_command)
+      {
+         pgmoneta_json_put(header, MANAGEMENT_ARGUMENT_COMMAND, (uintptr_t)translated_command, ValueString);
+      }
+
+      out_format = (int32_t)pgmoneta_json_get(header, MANAGEMENT_ARGUMENT_OUTPUT);
+      translated_out_format = translate_output_format(out_format);
+      if (translated_out_format)
+      {
+         pgmoneta_json_put(header, MANAGEMENT_ARGUMENT_OUTPUT, (uintptr_t)translated_out_format, ValueString);
+      }
+
+      free(translated_command);
+      free(translated_out_format);
+   }
+
+   // Translate the response
+   response = (struct json*)pgmoneta_json_get(j, MANAGEMENT_CATEGORY_RESPONSE);
+
+   if (response && command)
+   {
+      switch (command)
+      {
+         case MANAGEMENT_BACKUP:
+         case MANAGEMENT_RESTORE:
+         case MANAGEMENT_RETAIN:
+         case MANAGEMENT_EXPUNGE:
+         case MANAGEMENT_INFO:
+         case MANAGEMENT_ANNOTATE:
+            translate_backup_argument(response);
+            break;
+         case MANAGEMENT_LIST_BACKUP:
+            backups = (struct json*)pgmoneta_json_get(response, MANAGEMENT_ARGUMENT_BACKUPS);
+            backup_dq = (struct deque*)backups->elements;
+            // Iterate the backups
+            pgmoneta_deque_iterator_create(backup_dq, &backup_dq_it);
+            while (pgmoneta_deque_iterator_next(backup_dq_it))
+            {
+               backup = (struct json*)pgmoneta_value_data(backup_dq_it->value);
+               translate_backup_argument(backup);
+            }
+            pgmoneta_deque_iterator_destroy(backup_dq_it);
+            break;
+         case MANAGEMENT_STATUS_DETAILS:
+            servers = (struct json*)pgmoneta_json_get(response, MANAGEMENT_ARGUMENT_SERVERS);
+            server_dq = (struct deque*)servers->elements;
+            // Iterate server
+            pgmoneta_deque_iterator_create(server_dq, &server_dq_it);
+            while(pgmoneta_deque_iterator_next(server_dq_it))
+            {
+               backups = (struct json*)pgmoneta_json_get((struct json*)pgmoneta_value_data(server_dq_it->value), MANAGEMENT_ARGUMENT_BACKUPS);
+               backup_dq = (struct deque*)backups->elements;
+               // Iterate the backups
+               pgmoneta_deque_iterator_create(backup_dq, &backup_dq_it);
+               while (pgmoneta_deque_iterator_next(backup_dq_it))
+               {
+                  backup = (struct json*)pgmoneta_value_data(backup_dq_it->value);
+                  translate_backup_argument(backup);
+               }
+               pgmoneta_deque_iterator_destroy(backup_dq_it);
+            }
+            pgmoneta_deque_iterator_destroy(server_dq_it);
+            break;
+         default:
+            break;
+      }
+   }
+
+}
+
```

Comment:
static functions in `cli.c`

Reply:
So basically, move everything to `cli.c` and make static functions!

Reply:
Yup